src/services/Serialization.jl

```diff
@@ -30,6 +30,9 @@ function unpackVariable(dfg::G,
         unpackBigData::Bool=true)::DFGVariable where G <: AbstractDFG
     @debug "Unpacking variable:\r\n$packedProps"
     label = Symbol(packedProps["label"])
+    # Make sure that the timestamp is correctly formatted with subseconds
+    packedProps["timestamp"] = replace(packedProps["timestamp"], r":(\d)(\d)(Z|z|\+|-)" => s":\1\2.000\3")
```

Comment:
Is the correct format not enforced by my change in ln 11:
`props["timestamp"] = Dates.format(v.timestamp, "yyyy-mm-ddTHH:MM:SS.ssszzz")`

As far as I could tell this was only a bug in CGDFG and worked correctly with forcing the format as above

Reply:
11?

Reply:
Nope, see here: https://github.com/JuliaRobotics/DistributedFactorGraphs.jl/issues/582#issuecomment-671976222

Reply:
Not the same thing

Reply:
As far as I could tell it was working in v0.9, just not in the cloud serialization, which is separate.